storage/cloudspanner/spanner.sdl

```diff
@@ -29,10 +29,7 @@ CREATE TABLE TreeHeads(
   RootSignature           BYTES(1024) NOT NULL,
   TreeRevision            INT64 NOT NULL,
   TreeMetadata            BYTES(2097152),
-) PRIMARY KEY(TreeID, TimestampNanos DESC);
-
-CREATE UNIQUE INDEX TreeRevisionIndex
-  ON TreeHeads(TreeID, TreeRevision DESC);
+) PRIMARY KEY(TreeID, TreeRevision DESC);
```

Comment:
Just for the release notes this looks like an incompatible change?

Reply:
Technically yes, but presumably nobody other than me has an instance of it :)

Reply:
Added to draft release notes in case.